packages/strapi-plugin-users-permissions/controllers/User.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/**
+ * User.js controller
+ *
+ * @description: A set of functions called "actions" for managing `User`.
+ */
+
+const _ = require('lodash');
+
+module.exports = {
+
+  /**
+   * Retrieve user records.
+   *
+   * @return {Object|Array}
+   */
+
+  find: async (ctx) => {
+    let data = await strapi.plugins['users-permissions'].services.user.fetchAll(ctx.query);
```

Comment:
`fetchAll` will always return an array so it could be better to use `map` to filter the keys. It also avoids checking if data is empty or not.


```js
const data = await strapi.plugins['users-permissions'].services.user.fetchAll(ctx.query)
  .map(user => _.omit(user.toJSON(), ['password', 'resetPasswordToken']));
```

Btw, I have a doubt with the `toJSON` method in the controller. It's related to Bookshelf, are you sure it also works with Mongoose or vice versa?